temba/channels/models.py

```diff
@@ -1105,6 +1129,48 @@ def success(cls, channel, msg, msg_status, start, request_method=None, request_u
                                   request_time=request_time)
 
     @classmethod
+    def send_fcm_message(cls, channel, msg, text):
+        from temba.msgs.models import Msg, WIRED
+        start = time.time()
+
+        api_key = channel.config.get(Channel.CONFIG_FCM_KEY) if Channel.CONFIG_FCM_KEY in channel.config else None
+
+        if api_key:
+            url = 'https://fcm.googleapis.com/fcm/send'
+            title = channel.config.get(Channel.CONFIG_FCM_TITLE)
+            data_message = {'type': 'rapidpro', 'title': title, 'message': text, 'message_id': msg.id}
+            payload = json.dumps({
+                'data': data_message,
+                'content_available': True,
+                'to': msg.urn_path,
+                'notification': {
```

Comment:
So if I understand this right, messages from RapidPro will always display as a notification, is that right?

Should that be an option when adding the channel as to whether you want notifications or pure data?